experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyTree.ts

```diff
@@ -276,6 +276,20 @@ export abstract class LazyTree<TSchema extends TreeSchema = TreeSchema>
 				);
 				return unsubscribeFromSubtreeChange;
 			}
+			case "beforeChange": {
+				const unsubscribeFromChildrenBeforeChange = this.#anchorNode.on(
```

Comment:
I liked the pattern because it makes it very clear by just looking at this code (no need to go to the definition for `on`) what is it exactly that's getting returned.